### DISCUSSION TITLE
Airflow Dag executions does not match with schedule

### DISCUSSION THREAD

**Discussion by harounben14** [apache/airflow#24074] [Q&A]
### Apache Airflow version

2.2.2

### What happened

Here is the schedule of my dag: 0 6 * * 1-5
It should run every day of the week (Monday to Friday)
However, the Monday execution never happens.

I tried to delete the dag and his metadata, but the issue still exists...

### What you think should happen instead

![Capture d’écran 2022-06-01 à 14 40 15](https://user-images.githubusercontent.com/97513091/171406600-e2a4aeae-346d-408f-9f8c-56503b2e8e8a.png)

### Operating System

Mac OS Monterey

### Deployment

Other Docker-based deployment

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Airflow runs at end of interval by default - not at the beginning. Your Monday run ends on Tuesday  at 6 am and then it is executed.

**Reply by harounben14** [reply]
Thanks for your answer! So what should I change in my schedule interval to make my dag run on Monday ?

**Reply by harounben14** [reply]
Something like "0 6 * * 0-5"?

**Reply by potiuk** [reply]
Well. Whatever your end-intervals are :)

**Comment by pingzh** [comment]
you can find more info: https://airflow.apache.org/docs/apache-airflow/2.2.2/concepts/scheduler.html

>If you run a DAG on a schedule_interval of one day, the run with data interval starting on 2019-11-21 triggers after 2019-11-21T23:59.
>
> Let’s Repeat That, the scheduler runs your job one schedule_interval AFTER the start date, at the END of the interval.
